frame/contracts/src/wasm/mod.rs

```diff
@@ -375,8 +376,8 @@ mod tests {
 				)
 			)
 		}
-		fn get_weight_price(&self) -> BalanceOf<Self::T> {
-			1312_u32.into()
+		fn get_weight_price(&self, weight: Weight) -> BalanceOf<Self::T> {
+			(weight * 1312).into()
```

Comment:
why not `saturating_mul`?

Reply:
I didn't bother because it's test code. It does not hurt, though. I switched to `saturating_mul`.

Reply:
Ok, I didn't notice that :)